plugins/ecs-bridge/engine/configureveth.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't see a big value with doing that. It's not that big a string and is composed of 2 distinct parts.